migrations/20231210123554_init.sql

```diff
@@ -0,0 +1,236 @@
+    finished_at DATE,
+    priority    "TaskPriority" NOT NULL,
+    accepted    "AcceptanceStatus" NOT NULL,
+    -------------------------------------------------------
+    created_at  TIMESTAMP NOT NULL DEFAULT now(),
+    edited_at   TIMESTAMP NOT NULL DEFAULT now(),
+    deleted_at  TIMESTAMP,
+    -------------------------------------------------------
+    event_id    UUID,
+    creator_id  UUID,
+    -------------------------------------------------------
+    FOREIGN KEY (event_id) REFERENCES "Event" (id),
+    FOREIGN KEY (creator_id) REFERENCES "EventStaff" (id)
+);
+
+
+CREATE TABLE IF NOT EXISTS "TaskAsignee"
+(
+    status      "AcceptanceStatus" NOT NULL,
+    -------------------------------------------------------
+    created_at  TIMESTAMP NOT NULL DEFAULT now(),
+    edited_at   TIMESTAMP NOT NULL DEFAULT now(),
+    deleted_at  TIMESTAMP,
+    -------------------------------------------------------
+    task_id     UUID,
+    user_id     UUID,
+    -------------------------------------------------------
+    FOREIGN KEY (task_id) REFERENCES "Task" (id),
+    FOREIGN KEY (user_id) REFERENCES "User" (id)
+);
+
+
+CREATE TABLE IF NOT EXISTS "Comment"
+(
+    id          UUID PRIMARY KEY DEFAULT gen_random_uuid(),
+    -------------------------------------------------------
+    content     TEXT NOT NULL,
+    -------------------------------------------------------
+    created_at  TIMESTAMP NOT NULL DEFAULT now(),
+    edited_at   TIMESTAMP NOT NULL DEFAULT now(),
+    deleted_at  TIMESTAMP,
+    -------------------------------------------------------
+    event_id    UUID,
+    author_id   UUID,
+    task_id     UUID,
+    -------------------------------------------------------
+    FOREIGN KEY (event_id) REFERENCES "Event" (id),
+    FOREIGN KEY (author_id) REFERENCES "User" (id),
+    FOREIGN KEY (task_id) REFERENCES "Task" (id)
+);
+
+
+CREATE TABLE IF NOT EXISTS "AssociatedCompany"
+(
+    type        "Association" NOT NULL,
+    -------------------------------------------------------
+    created_at  TIMESTAMP NOT NULL DEFAULT now(),
+    edited_at   TIMESTAMP NOT NULL DEFAULT now(),
+    deleted_at  TIMESTAMP,
+    -------------------------------------------------------
+    company_id  UUID,
+    event_id    UUID,
+    -------------------------------------------------------
+    FOREIGN KEY (company_id) REFERENCES "Company" (id),
+    FOREIGN KEY (event_id) REFERENCES "Event" (id)
+);
+
+
+CREATE TABLE IF NOT EXISTS "Adress"
+(
+    id          UUID PRIMARY KEY DEFAULT gen_random_uuid(),
+    -------------------------------------------------------
+    country     VARCHAR(255) NOT NULL,
+    region      VARCHAR(255) NOT NULL,
+    city        VARCHAR(255) NOT NULL,
+    street      VARCHAR(255) NOT NULL,
+    number      VARCHAR(255) NOT NULL,
+    postal_code VARCHAR(255) NOT NULL
+);
+
+
+CREATE TABLE IF NOT EXISTS "Employment"
+(
+    hourly_rage FLOAT NOT NULL,
+    start_date  DATE NOT NULL,
+    end_date    DATE,
```

Comment:
It might be wise to have this not null as well. What we can do for contracts that don't have an end_date is put end_date to a huge date, like 9999.12.31. Having it set as null can affect some queries.